<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgScriptVar</class>
 <widget class="QDialog" name="DlgScriptVar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Variable</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="props">
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="varName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Åƒame</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="varID">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="varType"/>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="chkReadOnly">
        <property name="text">
         <string>Read only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="value">
     <property name="title">
      <string>Value</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="lblVec2">
        <property name="text">
         <string>Vec2</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="boolValueTrue">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>True</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="boolValueFalse">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>False</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblBool">
        <property name="text">
         <string>Boolean</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblString">
        <property name="text">
         <string>String</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblInteger">
        <property name="text">
         <string>Integer</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblFloat">
        <property name="text">
         <string>Float</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="strValue">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QDoubleSpinBox" name="vec2ValueX">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <double>-1000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="vec2ValueY">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="minimum">
           <double>-1000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="floatValue">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="intValue">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>-10000000</number>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAccept">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>varID</tabstop>
  <tabstop>varName</tabstop>
  <tabstop>varType</tabstop>
  <tabstop>chkReadOnly</tabstop>
  <tabstop>strValue</tabstop>
  <tabstop>intValue</tabstop>
  <tabstop>floatValue</tabstop>
  <tabstop>vec2ValueX</tabstop>
  <tabstop>vec2ValueY</tabstop>
  <tabstop>boolValueTrue</tabstop>
  <tabstop>boolValueFalse</tabstop>
  <tabstop>btnAccept</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
